rocsp/config/rocsp_config.go

```diff
@@ -138,3 +146,80 @@ func MakeReadClient(c *RedisConfig, clk clock.Clock) (*rocsp.Client, error) {
 	})
 	return rocsp.NewClient(rdb, timeout, clk), nil
 }
+
+// A ShortIDIssuer combines an issuance.Certificate with some fields necessary
+// to process OCSP responses: the subject name and the shortID.
+type ShortIDIssuer struct {
```

Comment:
If we don't need to store the raw subject (see comment on `FindIssuerByName`), then providing a ShortID would be the only purpose of the ShortIDIssuer type. At which point -- why not put that directly on `issuance.Certificate` and get rid of this type entirely.

And then if we're getting rid of this type, we can instead create a compound type (more similar to the CA's IssuerMaps https://github.com/letsencrypt/boulder/blob/main/ca/ca.go#L54) which has `FindIssuerByID` and `FindIssuerByKeyHash` as methods instead of freestanding functions.

Reply:
Yeah, I think collapsing ShortIDIssuer into issuance.Certificate is a good idea, although it does raise the issue that _some_ instantiations of issuance.Certificate might not have it (until/unless we convert all users of issuance.Certificate to load the short IDs).

I'd kind of like to treat this as a followup refactoring - this change is pulling `ShortIDIssuer` out of rocsp-tool for reuse without dramatically changing it.

Reply:
Ah right, I'd already forgotten that the short IDs are hand-curated, not autogenerated. Yep, agreed that this should be a followup at best, since propagating the ShortIDIssuer config map through all the other services seems like not my favorite thing in the world.